spec/support/analytics_helper.rb

```diff
@@ -1,7 +1,14 @@
 module AnalyticsHelper
-  def stub_analytics
-    controller.analytics = FakeAnalytics.new
-    @analytics = controller.analytics
+  def stub_analytics(user: nil)
```

Comment:
I think these changes are causing a lot of existing tests to fail, I'll flip this back to draft and investigate on Monday.

Reply:
> I think these changes are causing a lot of existing tests to fail, I'll flip this back to draft and investigate on Monday.

Fixed in 2178314 and marked ready for review.